module/addRemove.js

```diff
@@ -0,0 +1,139 @@
+let todos = JSON.parse(localStorage.getItem('todos')) || [];
+let editing = -1;
+const toDoInput = document.querySelector('#new-list');
+const todoListElement = document.querySelector('.to-do-list');
+
+const saveToDos = () => {
+  const todovalue = toDoInput.value;
+  const isDuplicate = todos.some((todo) => todo.value === todovalue);
+  if (todovalue === '') {
+    notifyUser('To do list can not be empty!');
+  } else if (isDuplicate) {
+    notifyUser('it is already in you list. please add another.');
+  } else {
+    if (editing >= 0) {
+      todos = todos.map((todo, index) => {
+        return {
+          value: index === editing ? todovalue : todo.value,
+          completed: todo.completed,
+          index: index,
+        };
+      });
+      editing = -1;
+    } else {
+      todos.push({
+        value: todovalue,
+        completed: false,
+        index: todos.length,
+      });
+    }
+    toDoInput.value = '';
+  }
+};
+
+const displayTodos = () => {
+  if (todos.length === 0) {
+    todoListElement.innerHTML =
+      '<center>Nothing to do yet! please add to do list </center>';
+    return;
+  }
+  todoListElement.innerHTML = '';
+  todos.forEach((todo, index) => {
+    todoListElement.innerHTML += `
+    <div class="todo" id="${index}">
+    <input type="checkbox" class="check-box" />
```

Comment:
- [ ] Good job so far :+1: Your project is really nice but it left with one main thing that your project would be better if you implement. I notice your the status of the `completed` get updated in the localStorage when I check the checkbox :100:  but why don't you check if the status of the `completed` while looping through is true or false, then base on that to update the UI? Unless when I refresh the page the status of `completed` is true in the localStorage, it remains true in the localStorage while the checkbox is unchecked on the UI.
![Screenshot from 2023-01-26 21-17-55](https://user-images.githubusercontent.com/92303891/214952554-7734808b-896f-44ee-ba9e-5519f0857007.png)